evap/evaluation/models.py

```diff
@@ -173,18 +173,18 @@ class Questionnaire(models.Model):
     objects = QuestionnaireManager()
 
     class Meta:
-        ordering = ('type', 'order', 'name_de')
+        ordering = ('type', 'order', 'pk')
         verbose_name = _("questionnaire")
         verbose_name_plural = _("questionnaires")
 
     def __str__(self):
         return self.name
 
     def __lt__(self, other):
-        return (self.type, self.order, self.name_de) < (other.type, other.order, self.name_de)
+        return (self.type, self.order, self.pk) < (other.type, other.order, other.pk)
```

Comment:
what did we do here? 🤦‍♂

Reply:
🤷‍♂ 

(no i definitely didn't git blame that)